rewards/df-challengedf.md

```diff
@@ -0,0 +1,47 @@
+---
+description: >-
+  Weekly prediction challenges for data scientists to compete, launch algorithms, and start earning. 🏆
+---
+
+# Win Competitions in Challenge DF
+
+<figure><img src="../.gitbook/assets/rewards/gif/eat-data.gif" alt=""><figcaption><p>Let's consume some data.</p></figcaption></figure>
+
+## Publish prediction models and earn rewards
+
+If you're eager to consume data and create valuable assets for publishing but unsure of where to begin, fear not! We've got you covered.
+
+Data Farming offers rewards to data scientists who participate in weekly challenges. We provide the rules and rewards, allowing you to focus on delivering high-quality results.  
+
+**Prizes:**
+1st Place - 2500 OCEAN
```

Comment:
Please make these a list. When they are loaded in gitbook, they are on the same row

Reply:
done